src/textual/drivers/linux_driver.py

```diff
@@ -248,6 +248,7 @@ def on_terminal_resize(signum, stack) -> None:
         self.flush()
         self._key_thread = Thread(target=self._run_input_thread)
         send_size_event()
+        self.write("\x1b]11;?\x07")  # Detect background color
```

Comment:
AFAIK some terminals respond with nothing when they see a sequence they don't recognize/support, i.e. if you send requests `<supported 1>;<unsupported 2>;<supported 3>;` the responses will be `<response 1>;<response 3>` only. If anything expects order guarantee for requests it may end up waiting forever. 
Is this case handled (I didn't do an in-depth review, just asking)?

Reply:
I try to handle this case gracefully by only parsing the RGB colors if the length is as expected. But, please take a good look at the code to ensure I do not have missed anything. I would also like to look into writing some unit tests for this, including the case you mention here